### PR TITLE
1.0.0

### PR DESCRIPTION
**Pull request by bakspace-itk**

### CODE REVIEW

File: robot_framework/process.py [hunk 1]
```diff
@@ -1,8 +1,142 @@
...
+        ws.append(row)
+
+    excel_buffer = BytesIO()
+    wb.save(excel_buffer)
+    excel_buffer.seek(0)
+
+    return excel_buffer
+
+
+def encrypt_data(cpr, first_name) -> str:
```
Comment (ghbm-itk) [inline robot_framework/process.py:81]: Missing hints

**Review by ghbm-itk — DISMISSED** [review]

File: robot_framework/process.py [hunk 2]
```diff
@@ -1,8 +1,142 @@
...
+    process_arguments = json.loads(orchestrator_connection.process_arguments)
+
+    # Prepare access to service platform
+    kombit_access = KombitAccess(process_arguments["service_cvr"], process_arguments["certificate_path"], True)
+
+    # Receive queue item list of people who weren't registered last time
+    queue_elements = orchestrator_connection.get_queue_elements(config.QUEUE_NAME)
+
+    # Find current list of people with unknown address and get their registration status for Digital Post
+    query = "SELECT * FROM [DWH].[Mart].[AdresseAktuel] WHERE Vejkode = 9901 AND Myndighed = 751"
```
[the remark this answers is not in the document]
Comment (bakspace-itk) [reply robot_framework/process.py:32]: Because it is more readable.

**Review by bakspace-itk — COMMENTED** [review]
Changes implemented

File: robot_framework/process.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ghbm-itk) [reply robot_framework/process.py:32]: !

**Review by ghbm-itk — CHANGES_REQUESTED** [review]

**Review by ghbm-itk — APPROVED** [review]

**Pull request merged**